test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -638,10 +639,10 @@ func TestProjectWithNetworkPeering(t *testing.T) {
 		t.Cleanup(func() {
 			deleteAllNetworkPeers(t, cliPath, generator.projectID, gcpEntity)
 		})
-		var createdNetworkPeer mongodbatlas.Peer
+		var createdNetworkPeer mongodbatlasv2.GCPPeerVpc
 		err = json.Unmarshal(resp, &createdNetworkPeer)
 		require.NoError(t, err)
-		expectedProject.Spec.NetworkPeers[0].ContainerID = createdNetworkPeer.ContainerID
+		expectedProject.Spec.NetworkPeers[0].ContainerID = createdNetworkPeer.ContainerId
```

Comment:
Can be a separate PR, but would it be worthwhile or overkill to have e2e tests for each of GCP/Azure/AWS @gssbzn ?